pkg/frontend/adminactions/resources_list.go

```diff
@@ -20,7 +20,7 @@ import (
 func (a *azureActions) ResourcesList(ctx context.Context) ([]byte, error) {
 	clusterRGName := stringutils.LastTokenByte(a.oc.Properties.ClusterProfile.ResourceGroupID, '/')
 
-	resources, err := a.resources.ListByResourceGroup(ctx, clusterRGName, "", "", nil)
+	resources, err := a.resources.ListByResourceGroup(ctx, clusterRGName, "$filter=resourceType ne 'Microsoft.Compute/snapshots'", "", nil)
```

Comment:
do you know what the memory profile looks like before and after this change with varying sizes of snapshots?
10 snapshots?
100 snapshots?
1000 snapshots?

Reply:
I'd either automate via powershell or manually go run the list azure resources against a subset of clusters in each region and gather some data on the type and count of additional resources.

Reply:
@bryanro92 I'm confused, this change filters out all snapshots from the result.

Reply:
Right, sorry for the confusion.

My first comment/ ask is how much benefit is that giving us in terms of memory and on varying scales.
Without this change, what is the memory profile of a list azure resources on a cluster with 10, 100, or 1000 snapshots (as examples). With this change - what is the memory profile afterwards? 

My second ask is to go collect data from clusters "in the wild", and quantify what data this action returns, and be able to identify from those results additional candidates of resources to filter.

Reply:
@bryanro92 It would also matter which types of resources the customer has spun up in the cluster resource group.  

It might help for us to look at our deny assignment and see which types of resources customers can create within the cluster resource group.